### DISCUSSION TITLE
WebSocket Connection to wss://... Failed, Socket error changed endpoint to wss://...

### DISCUSSION THREAD

**Discussion by RathodDeven** [mozilla/hubs#5073] [Hubs Cloud Questions]
I have successfully deployed hubs cloud, and it was working fine  until now. In console, it is showing socket endpoint changed error. 
Why is this happening all of a sudden? How to solve this ?
Here is log from AWS EC2 instance 
![image](https://user-images.githubusercontent.com/69627223/152095478-0cf5477f-86a8-4a5d-9e6d-d61d4ca99910.png)
and here is console log
<img width="364" alt="image" src="https://user-images.githubusercontent.com/69627223/152095616-14b89c3f-6c33-4893-8123-cba1d94a7fe8.png">

**Comment by alfainfotech** [comment]
I'm getting same 'Web Socket error... it worked fine for few days and all of sudden. Cant get to ADMIN not can do any changes... stuck up !!!

**Comment by pattersonbl2** [comment]
Would this be a custom client?  After doing light research, it could be a  cross-origin resource error.  I would start troubleshooting around this topic.

**Reply by alfainfotech** [reply]
FYI: Yes this is custom client on AWs. 1st time I installed after few days I got error, after trying and not able to find any resource, I deleted the stack and re-installed. Again after few says same thing... and again I have to delete and re-installed brand new fresh stack. So far its been 4 days and working fine, but not sure when same error will take over again. 
Plus I dont see I saw any CORS error on console. But you can keep an eye... Thx for follow up.

**Comment by jbshin-gemiso** [comment]
I also have the same issue. A custom client was being developed, but it was not applied to the hubs cloud.
No workaround was found. As in the case above, it suddenly happened to me while it was working fine.
